go/src/koding/db/mongodb/modelhelper/group.go

```diff
@@ -142,3 +143,17 @@ func CreateGroup(m *models.Group) error {
 
 	return Mongo.Run(GroupsCollectionName, query)
 }
+
+func MakeAdmin(accountId, groupId bson.ObjectId) error {
+	r := &models.Relationship{
+		Id:         bson.NewObjectId(),
+		TargetId:   accountId,
+		TargetName: "JAccount",
+		SourceId:   groupId,
+		SourceName: "JGroup",
+		As:         "admin",
+		TimeStamp:  time.Now().UTC(),
```

Comment:
Here time in UTC is used, in `CalculateAndApplyDeletedMembers` - in local timezone time. Not sure if intentional.